java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -372,6 +386,157 @@ public void computeResult_resultAlwaysInsideProvidedBoundaries() {
     }
   }
 
+  /** Calling computeConfidenceInterval before noised mean is computed throws an exception. */
+  @Test
+  public void computeConfidenceInterval_calledBeforeResult() {
+    assertThrows(IllegalStateException.class, () -> mean.computeConfidenceInterval(ALPHA));
+  }
+
+  @Test
+  public void computeConfidenceInterval_callsNoiseComputeConfidenceIntervalCorrectly() {
```

Comment:
callsNoiseAndComputeConfidenceIntervalCorrectly

Reply:
It should be "calls**Noise's**ComputeConfidenceIntervalCorrectly". The suggested name seems to imply that it callsNoise and actually computes confidence interval correctly or that it verifies addNoise(..) function call and computeConfidenceInterval(..) function call.

Reply:
Oh, I am sorry, I missed that on the line 417 you call computeConfidenceInterval and not addNoise.
Maybe just callsNoiseCorrectly?

since when we verify addNoise in other tests we don't specify what exactly it calls, there are many other methods.